src/StackAdmin/Azs.Backup.Admin/Module/Azs.Backup.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Set-AzsBackupShare.ps1

```diff
@@ -73,6 +73,7 @@ function Set-AzsBackupShare {
         [Parameter(Mandatory = $true, ParameterSetName = 'Update')]
         [ValidateNotNullOrEmpty()]
         [System.String]
+        [Alias("Path")]
```

Comment:
Use this alias to align with the old parameter name in AZURESTACK-tools. What's more, the property name in API is also "Path"